ext/readme.md

```diff
@@ -52,14 +52,14 @@ ext | `pysrt` | [1.1.0](https://pypi.org/project/pysrt/1.1.0/) | `subliminal` |
 ext | `python-dateutil` | [2.8.0](https://pypi.org/project/python-dateutil/2.8.0/) | **`medusa`**, `guessit`, `imdbpie`, `tvdbapiv2` | Module: `dateutil`
 ext | `python-twitter` | [3.5](https://pypi.org/project/python-twitter/3.5/) | **`medusa`** | Module: `twitter`
 ext | `pytz` | [2018.4](https://pypi.org/project/pytz/2018.4/) | `subliminal`, `tzlocal` | -
-**ext2 ext3** | `PyYAML` | [5.1](https://pypi.org/project/PyYAML/5.1/) | `knowit` | Module: `yaml`
+**ext2 ext3** | `PyYAML` | [5.1](https://pypi.org/project/PyYAML/5.1/) | `guessit`, `knowit` | Module: `yaml`
```

Comment:
@sharkykh was it supposed to also add guessit as dep here?

Reply:
Ooh I used medusa_vendor_tools btw

Reply:
Short answer, no.
Long answer, `guessit` has `PyYAML` listed as a dependency for tests. It shouldn't have added this here. Probably faulty logic on my part. That's why the result of the script should always be double-checked before committing.

Reply:
Should be fixed with `medusa_vendor_tools v0.2.3`.